core/templates/dev/head/profile/profile.html

```diff
@@ -17,37 +17,33 @@
 
 {% block content %}
   <div class="container">
+    
+    <div class="col-xs-12" style="height:50px;"></div>
     <div class="row">
-      <h1>Profile: {{user_username}}</h1>
-    </div>
 
-    <div class="row">
-      <div role="form" class="form-horizontal">
         <div class="form-group">
-          <label class="col-lg-2 col-md-2 col-sm-2">Picture</label>
-          <div class="col-lg-10 col-md-10 col-sm-10">
+          <div class="col-lg-2 col-md-2 col-sm-2">
             {% if user_pic_data_url %}
               <img src="{{user_pic_data_url}}" class="img-thumbnail">
             {% else %}
-              <img src="/images/general/no_profile_picture.png" class="img-thumbnail">
-            {% endif %}
-          </div>
-        </div>
-      </div>
-    </div>
-    <div class="row">
-      <div role="form" class="form-horizontal">
-        <div class="form-group">
-          <label class="col-lg-2 col-md-2 col-sm-2">Bio</label>
-          <div class="col-lg-10 col-md-10 col-sm-10">
-            {% if user_bio %}
-              {{user_bio}}
-            {% else %}
-              <em>This user has not supplied a bio yet.</em>
+              <img src="/images/general/no_profile_picture.png" class="no_profile_img-thumbnail">
```

Comment:
The bootstrop .img-thumbnail class sets an image size that is too small. The image should be 180x180.

when I try to change the style in line, it does not work. This is what I tried: 
 <img src="/images/general/no_profile_picture.png" class = "img-thumbnail" style ="height:180px width:180px">

It was only through setting up a class that I was able to change the image height and width.